src/test/munit/sftp-delete-directory-test-case.xml

```diff
@@ -12,8 +12,20 @@
         http://www.mulesoft.org/schema/mule/sftp http://www.mulesoft.org/schema/mule/sftp/current/mule-sftp.xsd
         http://www.mulesoft.org/schema/mule/java http://www.mulesoft.org/schema/mule/java/current/mule-java.xsd">
 
-    <munit:config name="sftp-delete-directory-test-case.xml"/>
-
+    <munit:config name="sftp-delete-directory-test-case.xml">
+        <munit:parameterizations>
+            <munit:parameterization name="config">
+                <munit:parameters>
+                    <munit:parameter propertyName="config" value="config"/>
```

Comment:
Please choose more significant names. Everything here is called `config`

Reply:
got it, actually, default configuration name is "config" for the property "config". So, will be able to change only the parameterization name.